assets/dev/scss/editor/preview/preview.scss

```diff
@@ -34,6 +36,24 @@ html.elementor-html {
 
 .elementor-edit-area-active {
 
+	--primary-color: #{ $editor-info };
+	--secondary-color: #{ darken($editor-info, 20%) };
+	--outline-color: var( --primary-color );
+
+	.e-container {
+		// TODO: Use variable when available - waiting for final color from the Design.
+		--primary-color: #0098C7;
+		--secondary-color: #{ darken(#0098C7, 10%) };
```

Comment:
Do we have the final colors?

Reply:
Nope.
Design said it's final for now.

Reply:
Please set it on the SCSS variables file

Reply:
Done